R/load_truth.R

```diff
@@ -221,49 +224,55 @@ load_truth <- function (truth_source = NULL,
         # load data from file path
         truth <- readr::read_csv(file_path) 
         
-        if (source == "ECDC"){
-          truth <- truth %>%
-            dplyr::rename(date = week_start)
-        }
-        
         truth <- truth %>%
-          # add inc_cum and death_case columns and rename date column
+          # add inc_cum and death_case columns 
           dplyr::mutate(model = paste0("Observed Data (",source,")"), 
-                        target_variable = target,
-                        date = as.Date(date)) %>%
-          dplyr::rename(target_end_date = date)
-        
+                        target_variable = target)
+        # ECDC case/death is in ISO weeks
+        if ("week_start" %in% names(truth)) {
+          truth <- truth %>%
+            dplyr::mutate(week_start = as.Date(week_start),
+                          target_end_date = week_start + 5) %>% # to match epiweek
```

Comment:
With this change, I think it might break the unit test [here](https://github.com/reichlab/covidHubUtils/blob/2604288565dc0a49e2e4d3feb1446b8950a70af0/tests/testthat/test-load_truth.R#L183). This is checking if all `target_end_dates` from `"ECDC"` `truth_source` are Mondays. We should change it to Saturdays.